Tools-Override/CloudTest.targets

```diff
@@ -279,10 +279,23 @@
   <Target Name="CreateFuncTestListJson"
           DependsOnTargets="CreateAzureStorage"
           Condition="'$(Performance)' != 'true' or '$(FuncTestsDisabled)' != 'true'">
-    <!-- create item group of functional tests -->
-    <CreateItem Include="$(ForUploadList)">
-      <Output TaskParameter="Include" ItemName="FunctionalTest"/>
-    </CreateItem>
+    <!-- create item group of functional tests 
+         Due to recent refactoring, calculating the right paths for Uris is challenging, so just reuse the previous
+         include approach to get a full list. 
+         This allows us to use RecursiveDir as the path to the zip instead of guessing and generating it again.
+    -->
+    <ItemGroup>
+      <FunctionalTest Include="$(TestArchivesRoot)**/*.zip" >
+        <RelativeBlobPath>$(Platform)$(ConfigurationGroup)/Tests/$([System.String]::Copy('%(RecursiveDir)').Replace('\', '/'))%(Filename)%(Extension)</RelativeBlobPath>
+      </FunctionalTest>
+      <FunctionalTest Include="$(AnyOSTestArchivesRoot)**/*.zip" >
+        <RelativeBlobPath>$(Platform)$(ConfigurationGroup)/Tests/$([System.String]::Copy('%(RecursiveDir)').Replace('\', '/'))%(Filename)%(Extension)</RelativeBlobPath>
+      </FunctionalTest>
+      <!-- Only include Unix folders if supported by the current target OS -->
+      <FunctionalTest Condition="'$(TargetsUnix)' == 'true'" Include="$(UnixTestArchivesRoot)**/*.zip" >
+        <RelativeBlobPath>$(Platform)$(ConfigurationGroup)/Tests/$([System.String]::Copy('%(RecursiveDir)').Replace('\', '/'))%(Filename)%(Extension)</RelativeBlobPath>
+      </FunctionalTest>
+    </ItemGroup>
```

Comment:
good point, actually with the current way I'm doing this I can just ignore that metadata, removed it.